tests/integration/test_integration.py

```diff
@@ -5,9 +5,13 @@
 
 import pytest
 
+from proxy.common.utils import get_available_port
 from proxy.common._compat import IS_WINDOWS  # noqa: WPS436
 
 
+PROXY_PY_PORT = get_available_port()
```

Comment:
It's best to use an ephemeral port. Otherwise, it's still possible to hit race conditions.

Reply:
Yep I would have, but situation here is a little different.  Proxy is not in embedded mode bu started from command line.  So we must then either parse port from logs or let proxy write the port somewhere or use pid file to infer the port  :).  I went for the easy solution for now.